src/deploy/functions/build.ts

```diff
@@ -586,3 +595,32 @@ function discoverTrigger(endpoint: Endpoint, region: string, r: Resolver): backe
   }
   assertExhaustive(endpoint);
 }
+
+function interpolateEventArcChannel(channel: string, projectId: string): string {
+  if (!channel.includes("/")) {
+    const location = api.functionsDefaultRegion;
```

Comment:
hmm part of me is surprised that this isn't `region` - shouldn't eventarc channel be in the same region as the function.